kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/RuntimeDataServiceBase.java

```diff
@@ -669,11 +674,19 @@ public ProcessInstanceUserTaskWithVariablesList queryUserTasksByVariables(String
         if (payload != null) {
             filter = marshallerHelper.unmarshal(payload, payloadType, SearchQueryFilterSpec.class);
         }
-        return convertToUserTaskWithVariablesList(advanceRuntimeDataService.queryUserTasksByVariables(convertToServiceApiQueryParam(filter.getAttributesQueryParams()),
+        if(filter.getOwnersQueryParam() != null) {
```

Comment:
just compute the query param and the execute the new api.
filter.getOwnersQueryParam != null ? filter.getOwnersQueryParam : QueryParam.all(owners)